micro_dl/utils/image_utils.py

```diff
@@ -76,10 +76,13 @@ def rescale_nd_image(input_volume, scale_factor):
 
 def crop2base(im, base=2):
     """
-    Crop image to nearest smaller factor of the base (usually 2)
+    Crop image to nearest smaller factor of the base (usually 2), assumes xyz
+    format, will work for zyx too but the x_shape, y_shape and z_shape will be
+    z_shape, y_shape and x_shape respectively
 
     :param nd.array im: Image
     :param int base: Base to use, typically 2
+    :param bool crop_z: crop along z dim, only for UNet3D
```

Comment:
It doesn't look like crop2base currently take crop_z as an input parameters, and it's currently only present in docstring not in code?

Reply:
good catch, had this at some point and deleted it!